### PR TITLE
Struct Creation

### PR DESCRIPTION
**Pull request by iosdevrashad**
Added two structs for Recipe model and Ingredients.

### CODE REVIEW

File: Food4Soul/Food4Soul/DataModel/RecipeModel.swift [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+//  Food4Soul
+//
+//  Created by Rashad Surratt on 9/20/22.
+//
+
+import Foundation
+import SwiftUI
+
+struct RecipeModel: Identifiable {
+
```
Comment (Eugeneberezin) [inline Food4Soul/Food4Soul/DataModel/RecipeModel.swift:12]: Please remove extra space

Comment (Eugeneberezin) [reply Food4Soul/Food4Soul/DataModel/RecipeModel.swift:12]: Line 12

**Review by Eugeneberezin — CHANGES_REQUESTED** [review]

**Review by Eugeneberezin — APPROVED** [review]
LGTM

**Comment by iosdevrashad** [comment]
Should keep the branch or should I delete it?

**Comment by Eugeneberezin** [comment]
> Should keep the branch or should I delete it?

Delete it please

**Pull request merged**